### ISSUE TITLE
"set-cookie" headers split at commas, breaking when "expires" is present

### ISSUE THREAD

**Issue by vwong** [marko-js/run#27]
Specifying the "expires" attribute when setting a cookie, emits two "set-cookie" headers instead of one single header with the designated expiry. More specifically, a header like this:

```
set-cookie: flavor=chocolate%20chip; Expires=Wed, 23 Feb 2022 17:17:53 GMT
```

Gets converted into this:
```
set-cookie: flavor=chocolate%20chip; Expires=Wed
set-cookie: 23 Feb 2022 17:17:53 GMT
```

It looks like the culprit is https://github.com/marko-js/run/blob/main/packages/run/src/adapter/middleware.ts#L185

**Comment by rturnq** [comment]
Yeah, I was worried about that. I was trying to avoid parsing the header value, but clearly more is needed here. It looks like the [date format used](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Date) by `expires` could be tested by a simple regex to determine if the comma found is part of a date or signifies the beginning of another cookie.

**Comment by rturnq** [comment]
Fixed by #26
